packages/paste-design-tokens/build.js

```diff
@@ -0,0 +1,34 @@
+const esbuild = require('esbuild');
+
+// Resolve paths to dist files in compiled assets
+const DesignTokenDistResolver = {
+  name: 'DesignTokenDistResolver',
+  setup(build) {
+    // get any paths to dist and replace then as they become local
```

Comment:
Can you expand on this comment please? I'm still unclear as to what this is trying to fix.

Reply:
So the javascript utilities this is trying to compile and place in the `dist/` folder to publish, are dependent on the compiled design token assets, which also live in the `dist/` folder. But before being compiled, the utils live outside of `dist/`.

Because the pre compiled Design Tokens are in yaml, we can't import those directly with a relative path, so we have to import the json they produce via the `dist/` compiled tokens.

Alls well and good locally, as the typescript compiles and doesn't complain when trying to run and build it as everything is local to the utils source, including the path to `./dist/what-ever.json`.

Problem comes when you distribute the utils with the design tokens in the `dist/` folder. The compiled utils now live _next to_ the design token JSON is needs, and trying to import them from `./dist/what-ever.json` won't work. All this does is remove the `dist/` and make the compile javascript have a relative path the json it needs that is now a sibling.

Reply:
Thanks